schema/src/lib.rs

```diff
@@ -316,28 +328,75 @@ impl<K: IntoSchema, V: IntoSchema> IntoSchema for BTreeMap<K, V> {
     }
 }
 
-impl<V: IntoSchema> IntoSchema for BTreeSet<V> {
+impl<K: IntoSchema> IntoSchema for BTreeSet<K> {
+    fn type_name() -> String {
+        format!("Vec<{}>", K::type_name())
+    }
+    fn schema(map: &mut MetaMap) {
+        map.entry(Self::type_name()).or_insert_with(|| {
+            Metadata::Vec(VecMeta {
+                ty: K::type_name(),
+                sorted: true,
+            })
+        });
+        if !map.contains_key(&K::type_name()) {
+            K::schema(map)
+        }
+    }
+}
+
+#[cfg(feature = "std")]
+#[allow(clippy::implicit_hasher)]
+impl<K: IntoSchema, V: IntoSchema> IntoSchema for HashMap<K, V> {
```

Comment:
I've added support for `HashMap`/`HashSet` which forced me to introduce `std` feature. If anyone thinks we'll never need those, I'd be happy to remove